Gemfile

```diff
@@ -1,10 +1,15 @@
-# If you have OpenSSL installed, we recommend updating
-# the following line to use "https"
-source "http://rubygems.org"
+# If you do not have OpenSSL installed, change
+# the following line to use "http://"
+source "https://rubygems.org"
+
+# Uncomment these gems if you are using Windows
+# gem "wdm", "~> 0.1.0", platforms: [:mswin, :mingw]
+# gem "tzinfo-data", platforms: [:mswin, :mingw, :jruby]
 
 gem "middleman", ">= 4.0.0"
```

Comment:
I'm usually not a fan of locking all of your gems, but in this case it does seem like a good idea since it isn't updated all that frequently. Done :+1: